mobile/src/AppContainer.js

```diff
@@ -49,6 +50,17 @@ class MarketplaceApp extends React.Component {
     }
   }
 
+  /* Check the clipboard for a referral code
+   */
+  checkReferral = async () => {
+    const clipData = await Clipboard.getString()
+    if (clipData && clipData.startsWith(REFERRAL_PREFIX)) {
+      const referralCode = clipData.slice(REFERRAL_PREFIX.length)
+      console.debug(`referral code found: ${referralCode}`)
+      this.props.setReferralCode(referralCode)
```

Comment:
Looks good, in some other similar code we have we clear the clipboard, but probably not any reason that is necessary